src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -8,16 +8,18 @@
 
 #include "DataStructures/Tensor/TypeAliases.hpp"
 
+namespace gr {
 /*!
  * \ingroup GeneralRelativityGroup
  * \brief Computes Christoffel symbol of the first kind from derivative of
  * metric
  *
  * \details Computes Christoffel symbol \f$\Gamma_{abc}\f$ as:
- * \f$ \Gamma_{cab} = 1/2 ( \partial_a g_{bc} + \partial_b g_{ac} -  \partial_c
- *    g_{ab}) \f$
+ * \f$ \Gamma_{cab} = \frac{1}/{2} ( \partial_a g_{bc} + \partial_b g_{ac}
```

Comment:
Use exactly one of `\frac` or `/`.

Reply:
Good catch. Thanks! This is fixed and squashed.